sound/x86/intel_hdmi_audio.c

```diff
@@ -1802,8 +1804,11 @@ static int __hdmi_lpe_audio_probe(struct platform_device *pdev)
 	pdata->notify_audio_lpe = notify_audio_lpe;
 	spin_unlock_irq(&pdata->lpe_audio_slock);
 
+	pm_runtime_set_autosuspend_delay(&pdev->dev, INTEL_HDMI_AUDIO_SUSPEND_DELAY_MS);
```

Comment:
I have really no idea about this patch. Maybe should be a separate patch and not in this series?

Reply:
yes, this would not be an ASoC patch and will be send separately. I removed the earlier HDaudio patches which don't work at all with pm_runtime_resume_and_get(), so now these Atom HDMI patches seem a bit out of place indeed.